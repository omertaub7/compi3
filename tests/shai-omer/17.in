// byte overflow
void main() {
	byte x = 64b;
	byte y = 128b;
	byte z = x * 2b + y;
	printi(z); // 0
	y = y + (y - 1b);
	printi(y); // 255
	byte w = y + 5b;
	printi(w); // 4
	int u = y + 5;
	printi(u);	// 260
}