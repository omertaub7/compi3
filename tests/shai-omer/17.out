0
255
4
260
